### ISSUE TITLE
Frontend redesign suggestions

### ISSUE THREAD

**Issue by Privat33r-dev** [play-with-docker/play-with-docker.github.io#259]
## Issue
Current design feels outdated and uses outdated stack.

## Suggestions
- [ ] Remove quiz carousel #258
- [ ] Remove bootstrap. Use tailwind (requires webpack) or even native CSS with flexboxes.
- [ ] Remove jQuery. After removing bootstrap.min.js, the only thing left would be `resizeable`, which can be replaced by [this](https://github.com/nathancahill/split/tree/master/packages/splitjs) tiny package. It can even be done without webpack, using just `<script src="https://cdnjs.cloudflare.com/ajax/libs/split.js/1.6.0/split.min.js"></script>`, [docs](https://github.com/nathancahill/split/tree/master/packages/splitjs#documentation) cover its usage.
- [ ] Add dark theme

**Comment by marcosnils** [comment]
> Current design feels outdated and uses outdated stack.

yes, I agree. We have a more recent design that we used for https://play-with-go.dev/intro-to-play-with-go-dev_go119_en/. The code is open source also https://github.com/play-with-go/play-with-go. 

I think we could bring some of those ideas to this website. WDYT?



> * Remove quiz carousel [Change quiz & question format #258](https://github.com/play-with-docker/play-with-docker.github.io/issues/258)

+1 to explore this


> * Remove bootstrap. Use tailwind (requires webpack) or even native CSS with flexboxes.

I like this as well. I'm unsure about the effort to make this happen

> * Remove jQuery. After removing bootstrap.min.js, the only thing left would be `resizeable`, which can be replaced by [this](https://github.com/nathancahill/split/tree/master/packages/splitjs) tiny package. It can even be done without webpack, using just `<script src="https://cdnjs.cloudflare.com/ajax/libs/split.js/1.6.0/split.min.js"></script>`, [docs](https://github.com/nathancahill/split/tree/master/packages/splitjs#documentation) cover its usage.

SGTM to explore also


> * Add dark theme

Play-with-go https://play-with-go.dev/intro-to-play-with-go-dev_go119_en/ has a dark theme :heart:
